src/Core/src/Platform/iOS/MauiWKWebView.cs

```diff
@@ -130,8 +130,20 @@ public void LoadUrl(string? url)
 		// It also has to be shared at the point you call init
 		public static WKWebViewConfiguration CreateConfiguration()
 		{
+			// By default, setting inline media playback to allowed, including autoplay
```

Comment:
@mkArtakMSFT @Eilon  @TanayParikh looking for BlazorWebView input... Given this explanation, would it make sense to setup the same defaults for Blazor's webview? Is this needed there?

Reply:
This looks good to me 😄